docker-compose/keycloak/Dockerfile

```diff
@@ -10,19 +10,39 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
 
-FROM jboss/keycloak:16.1.1
+#
+# Use builder to integrate custom provider
+#
+FROM quay.io/keycloak/keycloak:20.0.0 as builder
```

Comment:
They published 3 bugfix releases since 20.0.0 (current is 20.0.3)
https://quay.io/repository/keycloak/keycloak?tab=tags

we should rather use  `quay.io/keycloak/keycloak:20.0` as base image

Reply:
20.0.3 did not work for me, I tried, so I was happy to move to 20.0.0 and will wait for 21.0.0